SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -2025,8 +2025,8 @@ void CNEMOEulerSolver::BC_Inlet(CGeometry *geometry, CSolver **solution_containe
   su2double *V_domain = new su2double[nPrimVar];  su2double *V_inlet = new su2double[nPrimVar];
   su2double *Normal   = new su2double[nDim];
 
-  su2double Spec_Density[config->GetnSpecies()];
   nSpecies = config->GetnSpecies();
+  su2double *Spec_Density = new su2double[nSpecies];
```

Comment:
Don't you get a warning due to this variable being un-initialized? (that was the one I got)

Reply:
I don't get it, but seen the code it is clear that this is a valid warning. I'll fix in the sense that the warning does not appear anymore. However, it will not be working code.